w_common/lib/src/browser/disposable_browser.dart

```diff
@@ -232,27 +232,27 @@ class Disposable implements disposable_common.Disposable {
 
   @override
   StreamSubscription<T> listenToStream<T>(
-          Stream<T> stream, void onData(T event),
-          {Function onError, void onDone(), bool cancelOnError}) =>
+          Stream<T> stream, void onData(T event)?,
```

Comment:
`onData` should be non-nullable. We have a null check further in.
```suggestion
          Stream<T> stream, void onData(T event),
```

Reply:
Also on line 491.

Reply:
We can remove the null checks at the bottom of the file if we make everything passed in non-nullable.